README.md

```diff
@@ -1,5 +1,60 @@
-# FCC Test Suite Template
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
```

Comment:
you did a great job!
but there are few things to change

Reply:
- [ ] OPTIONAL: I think the summary of the review comments and you should use the "Review changes" Github feature. Kindly refer to this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) if you don't know how to do it. I think the previous TSE should have pointed this out and that is why I am making it OPTIONAL.